app/services/piv_cac_service.rb

```diff
@@ -61,26 +60,23 @@ def token_decoded(token)
     end
 
     def token_response(token)
-      # FUTURE: For consistency replace raw NET::HTTP with Faraday equivalent.
-      http = Net::HTTP.new(verify_token_uri.hostname, verify_token_uri.port)
-      http.use_ssl = verify_token_uri.scheme == 'https'
-      http.verify_mode = OpenSSL::SSL::VERIFY_NONE if FeatureManagement.identity_pki_local_dev?
-      http.start do |post|
-        post.request(decode_request(verify_token_uri, token))
+      # Assume ssl is off unless verify_token_uri uses https
+      ssl_config = false
+      if verify_token_uri.scheme == 'https'
+        ssl_config = { verify: !FeatureManagement.identity_pki_local_dev? }
       end
```

Comment:
wait a minute, we should only be allowing plain HTTP in local dev, so these can be combined right?

Reply:
Agreed. I was going to talk to Hooper about it after Ada standup. I erred on the side of replicating functionality as written because I didn't know what if any side effects could be introduced by changing that logic.